scripts/frag_an.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+DESC="""EPP diluting the concentration of samples in the fragment analyzer step"""
+
+from argparse import ArgumentParser
+from genologics.lims import Lims
+from genologics.config import BASEURI,USERNAME,PASSWORD
+from scilifelab_epps.epp import attach_file, EppLogger
+import sys
+import csv
+from genologics.entities import *
```

Comment:
As far as I can see Process is the only entity. Using asterix seems a bit excessive.